### PR TITLE
fix(plugin): exclude readonly member call expressions

### PR DESCRIPTION
**Pull request by Killusions**
This covers a previously unhandled corner case to prevent readonly member call expressions from being documented.

### CODE REVIEW

**Comment by Killusions** [comment]
@spike-rabbit

**Comment by Killusions** [comment]
@spike-rabbit Added a test

File: eslint-plugin-defaultvalue/tests/rule-tester.test.ts [hunk 1]
```diff
@@ -70,7 +70,12 @@ class Test {
   /** @internal */
   internal = 'internal';
 
+  private tempFunc: () => any = () => {};
+  private tempFuncObj = { sub: tempFunc } as const;
```
Comment (timowolf) [inline eslint-plugin-defaultvalue/tests/rule-tester.test.ts:74]: Not sure - but private properties are excluded anyway.

File: eslint-plugin-defaultvalue/lib/rules/default-value.ts [hunk 2]
```diff
@@ -194,8 +194,16 @@ const extractCallExpression = (
         // This will never have a defaultValue.
         return undefined;
       }
+      // Should not be documented.
+      if (readonly) {
+        return undefined;
+      }
```
Comment (timowolf) [inline eslint-plugin-defaultvalue/lib/rules/default-value.ts:200]: Please explain why you did this change and add it into the commit message. I do not really understand the reasoning.

**Review by timowolf — CHANGES_REQUESTED** [review]

File: eslint-plugin-defaultvalue/lib/rules/default-value.ts (hunk 2 above)
Comment (Killusions) [reply eslint-plugin-defaultvalue/lib/rules/default-value.ts:200]: @timowolf It says in the commit message: exclude readonly member call expressions

It was previously discussed to remove readonly expressions (and also done), this is just covering an edge case

**Review by spike-rabbit — APPROVED** [review]
